### PR TITLE
migrated to ui-shared

### PR DESCRIPTION
**Pull request by edewit**
Signed-off-by: Erik Jan de Wit <erikjan.dewit@gmail.com>

### CODE REVIEW

File: js/apps/admin-ui/src/realm-settings/ClientProfileForm.tsx [hunk 1]
```diff
@@ -220,216 +216,207 @@ export default function ClientProfileForm() {
...
+              }}
             />
-          </FormGroup>
-          <FormGroup label={t("description")} fieldId="kc-description">
-            <KeycloakTextArea
-              id="kc-description"
-              data-testid="client-profile-description"
-              isReadOnly={isGlobalProfile}
-              {...register("description")}
+            <TextControl
```
Comment (jonkoops) [inline js/apps/admin-ui/src/realm-settings/ClientProfileForm.tsx:230]: This should be a `TextAreaControl`.

File: js/apps/admin-ui/src/realm-settings/EmailTab.tsx [hunk 2]
```diff
@@ -98,301 +97,232 @@ export const RealmSettingsEmailTab = ({
...
+                    id="kc-enable-start-tls"
+                    data-testid="enable-start-tls"
+                    label={t("enableStartTLS")}
+                    isChecked={field.value === "true"}
+                    onChange={(value) => field.onChange("" + value)}
+                  />
+                )}
+              />
+            </FormGroup>
+            <SwitchControl
```
Comment (jonkoops) [inline js/apps/admin-ui/src/realm-settings/EmailTab.tsx:205]: Could this be a `DefaultSwitchControl`.

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: js/apps/admin-ui/src/realm-settings/EmailTab.tsx (hunk 2 above)
Comment (edewit) [reply js/apps/admin-ui/src/realm-settings/EmailTab.tsx:205]: No, the labels need to be "enabled" and "disabled" and not "on" and "off"

Comment (jonkoops) [reply js/apps/admin-ui/src/realm-settings/EmailTab.tsx:205]: Ah, missed that, consider this resolved.

**Review by jonkoops — DISMISSED** [review]
LGTM

**Review by jonkoops — DISMISSED** [review]

**Review by jonkoops — APPROVED** [review]

**Pull request merged**